en/docs/security-announcements/cve-justifications/2025/CVE-2025-48913.md

```diff
@@ -17,19 +17,20 @@ A vulnerability was reported in Apache CXF affecting the Java Message Service (J
 ### REPORTED PRODUCTS
 
 - WSO2 API Manager : 3.2.0, 3.2.1, 4.0.0, 4.1.0, 4.2.0, 4.3.0, 4.4.0, 4.5.0
+- WSO2 Identity Server : 5.10.0, 5.11.0, 6.0.0, 6.1.0, 7.0.0, 7.1.0
 
 ### WSO2 JUSTIFICATION
 
-WSO2 API Manager (APIM) does **not** make use of the Apache CXF component `org.apache.cxf:cxf-rt-transports-jms`, which is the only module affected by this vulnerability [^2][^3]. Therefore, the **vulnerable code path is not present** or exercised in the APIM runtime.
+The reported prodcuts do **not** make use of the Apache CXF component `org.apache.cxf:cxf-rt-transports-jms`, which is the only module affected by this vulnerability [^2][^3]. Therefore, the **vulnerable code path is not present** or exercised in the APIM or IS runtime.
 
 Some vulnerability scanners may incorrectly flag this issue due to the inclusion of other CXF components such as `org.apache.cxf:cxf-core`. However, the `cxf-core` library does **not** include the `org.apache.cxf:cxf-rt-transports-jms` component, which contains the vulnerable classes, and therefore is **not affected** by this vulnerability. According to the Snyk advisories, [^3] confirms that the vulnerability affects `org.apache.cxf:cxf-rt-transports-jms`, whereas [^4] clearly states that `org.apache.cxf:cxf-core` is **not** impacted. This further validates that the reported CVE does **not** apply to WSO2 API Manager. These detections are **false positives** resulting from indirect references to CXF dependencies.
 
-The vulnerability is addressed in **Apache CXF 3.6.8 and above**, which requires **JDK 11**. This fix is applicable to APIM versions **3.2.1, 4.2.0, and newer versions**, as these are based solely on JDK 11 and above. Applying this fix to APIM 4.1.0 and earlier versions is **not feasible**, since those versions support both JDK 8 and JDK 11. Using CXF 3.6.8 in such environments would cause server startup failures under JDK 8 due to the new JDK baseline requirement.
+The vulnerability is addressed in **Apache CXF 3.6.8 and above**, which requires **JDK 11**. This fix is applicable to APIM versions **3.2.1, 4.2.0, and newer versions**, and IS version **6.0.0 and newer versions**, as these are based solely on JDK 11 and above. Applying this fix to APIM 4.1.0 and earlier versions, and IS 5.11.0 and earlier versions, is **not feasible**, since those versions support both JDK 8 and JDK 11. Using CXF 3.6.8 in such environments would cause server startup failures under JDK 8 due to the new JDK baseline requirement.
```

Comment:
@ayomawdb IMO we don't need this paragraph at all now. Since this is a false positive detection. WDYT?